src/main/java/kr/sesac/aoao/server/dino/controller/DinoController.java

```diff
@@ -0,0 +1,33 @@
+package kr.sesac.aoao.server.dino.controller;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import kr.sesac.aoao.server.dino.controller.dto.GetUserDinoResponse;
+import kr.sesac.aoao.server.dino.service.DinoService;
+
+/**
+ * @since 2024.01.19
+ * @author 김은서
+ */
+
+@RestController
+public class DinoController {
+	private final DinoService dinoService;
+	@Autowired
+	public DinoController(DinoService dinoService){
+		this.dinoService = dinoService;
+	}
+
+
+	@GetMapping("/getUserDinoInfo")
+	public ResponseEntity<GetUserDinoResponse> getDinoInfo(@RequestBody Long userId) {
+		GetUserDinoResponse userDinoResponse = dinoService.getDinoInfo(userId);
+		return ResponseEntity.ok(userDinoResponse);
+	}
```

Comment:
```suggestion
	@GetMapping("/dinos")
	public ResponseEntity<GetUserDinoResponse> getDinoInfo(Long userId) {
		GetUserDinoResponse userDinoResponse = dinoService.getDinoInfo(userId);
		return ResponseEntity.ok(userDinoResponse);
	}
```

저희가 `@RestController` 를 사용하고 있으니 RESTful 하게 설계해보면 어떨까요?? 만약 해당 API 가 '나의 공룡 정보 조회' API 라면 위에 수정한 것처럼 반영할 수 있을것 같아요!! RESTful 한 설계라는 것을 간단하게 설명하자면, 1. 행위보다는 자원을 중심으로 나타내며, 2. 단수형 보다는 복수형으로 나타내는 것 이라고 할 수 있을 것 같습니다 ㅎㅎ

또한 제가 기억하기로 `@RequestBody` 는 생략이 가능한 것으로 알고 있는데 이 부분은 확인 후 반영 부탁드립니다!!